api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/reshaping/Cropping2D.kt

```diff
@@ -5,103 +5,63 @@
 
 package org.jetbrains.kotlinx.dl.api.core.layer.reshaping
 
-import org.jetbrains.kotlinx.dl.api.core.KGraph
-import org.jetbrains.kotlinx.dl.api.core.layer.Layer
-import org.jetbrains.kotlinx.dl.api.core.shape.TensorShape
 import org.tensorflow.Operand
 import org.tensorflow.Shape
 import org.tensorflow.op.Ops
 
 /**
- * Cropping layer for 2D input (e.g. picture).
- * It crops along spatial dimensions, i.e. height and width.
+ * Cropping layer for 2D data (e.g. images)
  *
- * Input shape: 4D tensor with shape `(batch_size, rows, cols, channels)`
```

Comment:
Please, return the input/output shape specification and if possible it for Cropping1D/3D for consistency. It's added in a minor number of layers, but I'm going to add into every layer specification